### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
Generated by https://github.com/jsoref/spelling `f`; to maintain your repo, please consider `fchurn`
Found in jsoref/linguist/pull/1

### CODE REVIEW

File: res/ChartAddIns/Column Operations [hunk 1]
```diff
@@ -13,11 +13,11 @@ if (NON_EMPTY_SELECTION)
 		counter = counter + 1;
 	}
 
-	column_stensil = {different_columns,1};
+	column_stencil = {different_columns,1};
```
Comment (jsoref) [inline res/ChartAddIns/Column Operations]: I don't know if this is right, but I hope so

File: res/TemplateBatchFiles/PARRIS.bf [hunk 2]
```diff
@@ -251,7 +251,7 @@ if (MGGYChoice<0)
 
 ChoiceList (nucModelChoice, "Nucleotide Rate Matrix Options",1,SKIP_NONE,
 			"Jukes-Cantor",	 "Single rate type (all substitutions equally likely).",
-			"HKY85","Two rate types (transition/transverion ratio parameter kappa).",
+			"HKY85","Two rate types (transition/transversion ratio parameter kappa).",
```
Comment (jsoref) [inline res/TemplateBatchFiles/PARRIS.bf]: I'm pretty sure this is the correct word (there are many misspellings)

File: src/core/dataset_filter.cpp [hunk 3]
```diff
@@ -539,7 +539,7 @@ void    _DataSetFilter::FilterDeletions(_SimpleList *theExc) {
         if (patterns_to_be_removed.countitems()) {
           /**
            
-              Deleting all instances of delected patterns; also need to reindex 
+              Deleting all instances of selected patterns; also need to reindex
```
Comment (jsoref) [inline src/core/dataset_filter.cpp]: I assume this is correct

**Comment by stevenweaver** [comment]
@spond - do you have any substantial updates on a branch? If so, we would rather defer this merge to avoid conflicts with your changes.

**Comment by stevenweaver** [comment]
This PR has become too out of date. Apologies.

**Comment by jsoref** [comment]
@stevenweaver: if i updated it, would you look? would it help if I split it up?

**Pull request closed without merging**